app/models/archival_media_bag_parser.rb

```diff
@@ -81,7 +81,7 @@ def pbcore_parsers
     def image_files
       @image_files ||=
         begin
-          path.join("data").each_child.select { |file| [".jpg"].include? file.extname }.map { |file| ImageFile.new(path: file) }
+          path.join("data").each_child.select { |file| [".jpg", ".JPG"].include? file.extname }.map { |file| ImageFile.new(path: file) }
```

Comment:
Any reason not to `downcase` the extension, so any mixture of capitalization would work?

Reply:
sure I can do it that way!